jbmc/src/java_bytecode/java_entry_point.cpp

```diff
@@ -537,6 +537,7 @@ main_function_resultt get_main_symbol(
       symbol != nullptr,
       "resolve_friendly_method_name should return a symbol-table identifier");
 
+    config.main_symbol = symbol;
```

Comment:
:question: The less we use config, the better, is this really the best way to achieve what you want?

Reply:
Though I see was already using it: https://github.com/diffblue/cbmc/pull/4180/commits/85348809d2b52d3230140511be42e45f3ae28732#diff-5c5e737bf6dd6d1aa2892f71af61a580L175

Reply:
Out of band discussion: Vojta will add some kind of utility to start with that allows deducing the entry point from a model (possibly the model show _know_ the entry point function?